src/Components/SearchBar/index.jsx

```diff
@@ -1,39 +1,45 @@
-import React, { useState } from "react";
-import Suggestions from "../Suggestions";
-// import Select from "react-select";
+import React from "react";
+import { Input } from "antd";
+import { Dropdown, Menu } from "antd";
 import "./SearchBar.scss";
+const { Search } = Input;
 
 const SearchBar = (props) => {
-  const [isSuggestionsVisible, setSuggestionsVisibility] = useState(false);
-  const showSuggestions = () => {
-    setSuggestionsVisibility(true);
-  };
-  const hideSuggestions = () => {
-    setSuggestionsVisibility(true);
-  };
   const onWordSelected = (word) => {
     props.onWordSelection(word);
     props.setSearchTerm(word);
-    setSuggestionsVisibility(false);
   };
+  let wordList = props.searchResult.map((wordDetails, index) => ({
+    label: wordDetails.word,
+    key: index,
+  }));
+
   return (
     <div className="search-area">
       <div className="search-bar">
-        <input
-          type="text"
-          placeholder="Search the word..."
-          onFocus={showSuggestions}
-          onBlur={hideSuggestions}
-          onChange={(e) => props.setSearchTerm(e.target.value)}
-          value={props.searchTerm}
-        />
+        <Dropdown
+          overlay={
+            <Menu
+              onClick={({ key }) => {
+                onWordSelected(wordList[key].label);
+              }}
+              items={wordList}
+            />
+          }
+          trigger={["click"]}
+          placement="bottom"
+        >
+          <Search
+            type="text"
+            placeholder="Search the word..."
+            onChange={(e) => props.setSearchTerm(e.target.value)}
+            value={props.searchTerm}
+            onSearch={(word) => {
+              onWordSelected(word);
+            }}
```

Comment:
Could be:
```suggestion
            onSearch={onWordSelected}
```

We can do this because

  ```
  onSearch={(param) => func(param)}
  ```
  and 
  ```
  onSearch={func}
  ```
  are equivalent as long as `func` and `onSearch` has the same function signature.